tests/Unit/Helpers/PointwiseFunctions/Hydro/EquationsOfState/TestHelpers.hpp

```diff
@@ -226,5 +226,13 @@ void check(EosType in_eos, const std::string& python_file_name,
                      member_args...);
 }
 /// @}
+
+/// Test that cloning is correct, and that the equality operator is implemented
+/// correctly
+template <class EosType>
+void test_get_clone(std::unique_ptr<EosType> in_eos) {
```

Comment:
[optional] Take a `const EosType&` (or probably better `const EquationOfState<...>&` with the function templated on the EOS template parameters) as the argument, so callers don't have to give up ownership of a `unique_ptr`.